libcodechecker/server/server.py

```diff
@@ -244,7 +245,8 @@ def do_GET(self):
                 # serve the product list.
                 LOG.debug("Serving product list as homepage.")
                 self.path = '/products.html'
-            else:
+            else:                            # The current request came through a
```

Comment:
This comment isn't a full sentence, and it makes zero sense.

Reply:
This was just a typo.